index.js

```diff
@@ -1,10 +1,18 @@
-/* eslint-disable @typescript-eslint/no-var-requires */
-const numWorkers = process.env.NUM_WORKERS || require('os').cpus().length;
+const start = async () => {
+	// Set the desired es version for downstream modules that support it
+	(await import('@balena/es-version')).set('es2022');
+
+	await import('./init.js');
+};
+
+const numWorkers =
+	parseInt(process.env.NUM_WORKERS ?? '0', 10) ||
+	(await import('os')).cpus().length;
```

Comment:
Q: I guess it wouldn't be that bad if we did a normal import for 'os' right, since it's a node thing and I guess it would be fast?

Reply:
I would expect so but also wanted to go for minimal changes